Documentation/api-reference.md

```diff
@@ -68,72 +71,189 @@ m.aepSdk = AdobeAEPSDKInit()
+ADB_CONSTANTS = AdobeAEPSDKConstants()
+
+configuration = {}
+configuration[ADB_CONSTANTS.CONFIGURATION.EDGE_CONFIG_ID] = "<YOUR_CONFIG_ID>"
+configuration[ADB_CONSTANTS.CONFIGURATION.EDGE_DOMAIN] = "<YOUR_DOMAIN_NAME>"
+
+' Set the default consent to pending and update the consent later based on user preference
+configuration[ADB_CONSTANTS.CONFIGURATION.CONSENT_DEFAULT] = {
+    "consents": {
+        "collect": {
+            "val": "p"
+        }
+    }
+}
+
+m.aepSdk.updateConfiguration(configuration)
+```
+
+The `EDGE_CONFIG_ID` value is presented as `Datastream ID` in the [Datastream details](https://experienceleague.adobe.com/docs/experience-platform/edge/datastreams/configure.html?lang=en#view-details) page.
+
+The `EDGE_DOMAIN` value is the first-party domain mapped to the Adobe-provisioned Edge Network domain. For more information, see this [documentation](https://developer.adobe.com/client-sdks/documentation/edge-network/#domain-configuration)
+
+### Configure default consent
+
+The default consent configuration determines how data collection consent is managed before invoking the [setConsent](#setconsent) API. This setting is particularly important to avoid unintentionally collecting data from users in regions where consent is required prior to data collection.
+
+By default, users are opted in for all purposes.
+
+#### Example: Setting default collect consent to pending before collecting actual user consent
+```brightscript
+ADB_CONSTANTS = AdobeAEPSDKConstants()
+
+configuration = {}
+
+configuration[ADB_CONSTANTS.CONFIGURATION.CONSENT_DEFAULT] = {
+    "consents": {
+        "collect": {
+            "val": "p"
+        }
+    }
+}
+
+m.aepSdk.updateConfiguration(configuration)
+```
+
+## Consent APIs:
+
+### setConsent
+
+Sends consent preferences to the Edge Network. For details on setting default consent before collecting user preferences, refer to the [Configure default consent](#configure-default-consent) section.
+
+> [!Important]
+> Please provide the entire payload with all the fields required by the Adobe 2.0 Standard.
+
+##### Syntax
+
+```brightscript
+setConsent: function(data as object) as void
+```
+
+- `@param data as object : an associative array that includes consent data to be sent to the edge network.`
+
+##### Example: setConsent using consent object following Adobe 2.0 Standard
+
+> [!Note]
+> The "time" field under "metadata" should contain ISO 8601 formatted with UTC(Z) time zone, millisecond precision date as shown below.
+
+> [!TIP]
+> Use the [`ToISOString("milliseconds")`](https://developer.roku.com/docs/references/brightscript/interfaces/ifdatetime.md#toisostringformat-as-string-as-string) to generate timestamp in the required format.
+
+```brightscript
+  collectConsentYes = {
+    "consent": [
+      {
+        "standard": "Adobe",
+        "version": "2.0",
+        "value": {
+          "metadata": {
+            ' pass timestamp in ISO format
+            "time" = "2023-10-03T17:23:04.443Z"
```

Comment:
"time" : ...

Reply:
Could you also provide a code snippet for generating the time in this format using the toISOString API?